modules/document_repository/README.md

```diff
@@ -0,0 +1,41 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides a centralized location for non-candidate documents that are relevant to the study (i.e. unfilled versions of instruments, publications, or manual of operations).
+
+## Intended Users
+
+ Given the appropriate permissions, LORIS users can view, download, upload, and delete files, as well as edit information pertaining to those files.
+ This module is used primarily by study coordinators/administrators who share study protocols and manuals, as well as data entry staff who consult these documents.
+
+## Scope
+
+ The document can only be seen by Users from the Site that the document is associated with.
+
+## Permissions
+
+ A user that has "document_repository_view" permission can view and upload files in Document Repository.
+ A user that has "document_repository_delete" permission can delete files in Document Repository.
+
+## Configurations
+
+- The Document Repository enables users to upload and organize project files of any type that can easily be viewable for users with appropriate permissions.Give full permissions to store documents on server. To give full permissions to store documents on the server: 
```

Comment:
I would split the first sentence in two:
"The Document Repository enables users to upload and organize project files of any type. These files can easily be viewable for users with appropriate permissions."

Also, remove duplicated sentence in **_bold-italic_** below:
"permissions.**_Give full permissions to store documents on server._** To give"